generator/src/main/resources/endpointDefinitions/SystemInformation.yaml

```diff
@@ -0,0 +1,18 @@
+className: [system, Information]
+endpointPath: system-information
+nameField: Version
+idField: Version
+generateGet: false
+generatePut: false
+generatePost: false
+generatePojo: false
+memberVariables:
+  - name: version
+    type: String
+  - name: build
+    type: String
+  - name: latestGlobalKeyResetTime
+    type: String
+  - name: patentNumbers
+    type: List
+    listType: String
```

Comment:
I wonder if it makes sense to have this file given that we are not actually using it to generate anything. Do you know if it works fine without this file?

Do you think it makes sense to have this file given it isn't actually doing anything? (I'm not sure if we have ever run into this before so this could set the precedent).

Reply:
I used this file to generate the system-information classes, but later on, I forgot to remove it. Thanks.